krew/krew.yaml

```diff
@@ -3,7 +3,7 @@ kind: Plugin
 metadata:
   name: kadalu
 spec:
-  version: "v0.8.7"
+  version: {{ .TagName }}
```

Comment:
- Will `.TagName` be populated by release-bot?

Reply:
I'm not sure, since I have not locally tested but followed documentation and looked at other plugin repos as an example. 
But I found this in "krew-release-bot" [TagName Code](https://github.com/rajatjindal/krew-release-bot/blob/master/vendor/github.com/google/go-github/v29/github/github-accessors.go/#L11143).
Not sure how it picks up the version